cnf-tests/testsuites/e2esuite/rps/rps.go

```diff
@@ -0,0 +1,268 @@
+			if err != nil {
+				discoverySuccessful, discoveryFailedReason = false, "Can not run tests in discovery mode. Failed to discover required resources."
+				return
+			}
+			profile = discovered.Profile
+			nodeSelector = nodes.SelectorUnion(nodeSelector, profile.Spec.NodeSelector)
+		} else {
+			err = performanceprofile.FindOrOverridePerformanceProfile(performanceProfileName, machineConfigPoolName)
+			Expect(err).ToNot(HaveOccurred())
+			err = client.Client.Get(context.TODO(), apitypes.NamespacedName{Name: performanceProfileName}, profile)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		By("CleanSriov...")
+		networks.CleanSriov(sriovclient)
+
+		By("CreateSriovPolicyAndNetwork...")
+		networks.CreateSriovPolicyAndNetwork(sriovclient, namespaces.SRIOVOperator, sriovNetworkName, "testresource", "")
+
+		By("Checking the SRIOV network-attachment-definition is ready")
+		Eventually(func() error {
+			nad := netattdefv1.NetworkAttachmentDefinition{}
+			objKey := apitypes.NamespacedName{
+				Namespace: namespaces.SRIOVOperator,
+				Name:      sriovNetworkName,
+			}
+			err := client.Client.Get(context.Background(), objKey, &nad)
+			return err
+		}, 2*time.Minute, 1*time.Second).Should(BeNil())
+	})
+	BeforeEach(func() {
+		if discovery.Enabled() && !discoverySuccessful {
+			Skip(discoveryFailedReason)
+		}
+	})
+	Context("rps configuration with performance profile applied", func() {
+		workerRTNodes, err := ntonodes.GetByLabels(profile.Spec.NodeSelector)
+		Expect(err).ToNot(HaveOccurred())
+		BeforeEach(func() {
+			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved == nil {
+				Skip("Test Skipped due nil Reserved cpus")
+			}
+		})
+		It("check RPS Mask is applied to at least one single rx queue on all veth interface", func() {
+			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
+				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
+				Skip("realTime Workload Hints is not enabled")
+			}
+			count := 0
+			expectedRPSCPUs, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, node := range workerRTNodes {
+				var vethInterfaces = []string{}
+				allInterfaces, err := ntonodes.GetNodeInterfaces(node)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(allInterfaces).ToNot(BeNil())
+				// collect all veth interfaces in a list
+				for _, iface := range allInterfaces {
+					if iface.Bridge == true && iface.Physical == false {
+						vethInterfaces = append(vethInterfaces, iface.Name)
+					}
+				}
+				//iterate over all the veth interfaces and
+				//check if at least on single rx-queue has rps mask
+				klog.Infof("%v", vethInterfaces)
+				for _, vethinterface := range vethInterfaces {
+					devicePath := fmt.Sprintf("%s/%s", "/rootfs/sys/devices/virtual/net", vethinterface)
```

Comment:
just use `devicePath := "/rootfs/sys/devices/virtual/net" + "/" + vethinterface

Reply:
or you can even use tha path package to create it nicely